src/App.js

```diff
@@ -1,36 +1,25 @@
 import React from 'react';
-import { Router, Link } from '@reach/router';
-
+import { Router } from '@reach/router';
 import Home from './Home/components/Home';
 import MakeListingDisplay from './MakeListingDisplay/components/MakeListingDisplay';
-
 import './App.css';
+import SignInPage from "./AuthenticationApp/MainComponents/SignInPage";
+import SignUpPage from "./AuthenticationApp/MainComponents/SignUpPage";
+import UserWrappedMain from "./AuthenticationApp/MainComponents/UserWrappedMain";
+import PostCommentsDisplay from './PostCommentsDisplay/components/PostCommentsDisplay';
 
 /** Component class to render the whole comments page */
-function App(props) {
+function App () {
     return (
-        <div>
-            <h1>Roomies</h1>
-
-            <nav>
-                <Link to="/">
-                    <button type="button">
-                        Home
-                    </button>
-                </Link>
-                <Link to="/makeListing">
-                    <button type="button">
-                        Make a Listing
-                    </button>
-                </Link>
-            </nav>
-
-            <Router>
-                <Home path="/" />
-                <MakeListingDisplay path="/makeListing" />
-            </Router>
-        </div>
-    );
+        <Router>
+            <UserWrappedMain path= "/" />
+            <SignInPage path= "signIn" />
+            <SignUpPage path= "signIn/signUp" />
+            <MakeListingDisplay path="makeListing" />
+            <PostCommentsDisplay path= "postComment" />
+            <Home path= "home" />
+        </Router>
```

Comment:
I just want to make sure that these changes don't fundamentally change what was displayed by this component before and only adds what you have.